frontend/appflowy_tauri/src/appflowy_app/utils/slate/text.ts

```diff
@@ -30,3 +42,163 @@ export function getInsertRangeBy(editor: Editor) {
     focus: { path: [0, lastIndex], offset: lastNode.text.length },
   };
 }
+
+/**
+ * check if the point is in the beginning of the editor
+ * @param editor
+ * @param at
+ */
+export function pointInBegin(editor: Editor, at: Location) {
+  const start = Editor.start(editor, at);
+  return Editor.before(editor, start) === undefined;
+}
+
+/**
+ * check if the point is in the end of the editor
+ * @param editor
+ * @param at
+ */
+export function pointInEnd(editor: Editor, at: Location) {
+  const end = Editor.end(editor, at);
+  return Editor.after(editor, end) === undefined;
+}
+
+/**
+ * get the selection of the beginning of the node
+ */
+export function getNodeBeginSelection(): TextSelection {
+  const point = {
+    path: [0, 0],
+    offset: 0,
+  };
```

Comment:
Is this code unused? Or do you intend to assign it to a value below?

`{ anchor: point, focus: point }`.

Reply:
It is not allowed, it will use the same obj